src/CommonLib/utility.cxx

```diff
@@ -606,4 +567,25 @@ UTILITY::FindDotNetFolders(
     } while (FindNextFileW(handle, &data));
 
     FindClose(handle);
-}
\ No newline at end of file
+}
+
+HANDLE
+UTILITY::CheckIfFileExists(STRU* struFile)
```

Comment:
the input can be "_In_ PCWSTR pszFilePath" instead of STRU* as you do not use STRU functions